Generator/OrmGenerator.php

```diff
@@ -56,7 +56,14 @@ public function generateMappingEntityFiles(OutputInterface $output, BundleMetada
                 $output->writeln(sprintf('   ~ <info>%s</info>', $fileName));
             } else {
                 $output->writeln(sprintf('   + <info>%s</info>', $fileName));
-                copy($src_file, $dest_file);
+
+                $mappingEntityTemplate = file_get_contents($src_file);
+
+                $string = Mustache::replace($mappingEntityTemplate, array(
+                    'namespace' => $bundleMetadata->getExtendedNamespace(),
+                ));
```

Comment:
We can't use DI without a BC break.

There is another class who is using this class without DI: https://github.com/sonata-project/SonataEasyExtendsBundle/blob/2.x/Generator/BundleGenerator.php#L81